### ISSUE TITLE
License missing!

### ISSUE THREAD

**Issue by arockenberger** [dh-tech/dh-tech.github.io#26]
Ihr Lieben :)

Greetings from Oslo

**Comment by jdamerow** [comment]
yes, we should do that and I am so sorry we never even responded to this issue!

**Comment by ColeDCrawford** [comment]
- Add license - apache 2 for code / cc-by for content
- Add content license to site / footer
